cookiesales.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+//Select the HTML element we will be modifing
+var displayResult = document.getElementById('tableResults');
+
+//Create our Object template
+function StoreLocations (storeName,hourlyMinCustomer,hourlyMaxCustomer,avegCookies) {
+  this.storeName = storeName;
+  this.hourlyMinCustomer = hourlyMinCustomer;
+  this.hourlyMaxCustomer = hourlyMaxCustomer;
+  this.avegCookies = avegCookies;
+  this.hourlyCookieSales = [];
+  this.storeHours = ['6am', '7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm','8pm'];
+}
+
+//Create first Object Method for StoreLocations, this generates a random number of customers
+StoreLocations.prototype.forcastRandomCustomerPerHour = function(){
+  return Math.ceil(Math.random() * (this.hourlyMaxCustomer - this.hourlyMinCustomer +1) + this.hourlyMinCustomer);
+};
+
+//create second Object Method to calculate number of cookies sold
+StoreLocations.prototype.genCookieSales = function() {
+  return Math.ceil(Math.random()*(this.avegCookies * this.forcastRandomCustomerPerHour()));
+};
+
+//function to generate number of cookies per hour and store them into an array
+StoreLocations.prototype.genHourNumberCookie = function(){
+  for(var i = 0; i < this.storeHours.length; i++){
+    this.hourlyCookieSales.push(this.genCookieSales());
+  }
+};
+
+//Create function to create a table for us
+StoreLocations.prototype.createtableRows = function() {
+
+  var rowHours, rowHourDisplay, rowStoreName;
```

Comment:
While this is perfectly valid JavaScript, since rowHours and rowHourName are immediately assigned following this line you can declare the var in those lines to save retyping the names vs retyping 'var'..